include/tensor_filter_custom.h

```diff
@@ -101,7 +101,6 @@ typedef int (*NNS_custom_invoke) (void *private_data,
  * @param[in] prop GstTensorFilter's property values. Do not change its values.
  * @param[in] input pointer to input tensor, size = dim1 x dim2 x dim3 x dim4 x typesize, allocated by caller
```

Comment:
how about changing the description about input, like *NNS_custom_invoke above.
@param[in] input The array of input tensors, each tensor size = dim1 x dim2 x dim3 x dim4 x typesize, allocated by caller

Reply:
OK. I will